test/fixture/recipe/deploy.php

```diff
@@ -38,6 +38,7 @@
 
 desc('Deploy your project');
 task('deploy', [
+    'check_for_known_bugs',
```

Comment:
Also it's redundant to check it on every deploy. 
Maybe call in `doctor` and mention in docs what users should run this task to check errors.